.internal-ci/helm/fog-ingest/templates/fog-ingest-statefulset.yaml

```diff
@@ -21,6 +21,28 @@ spec:
         app: fog-ingest
         {{- include "fogIngest.labels" . | nindent 8 }}
     spec:
+      # Try to balance pods across zones
+      topologySpreadConstraints:
+      - topologyKey: topology.kubernetes.io/zone
+        maxSkew: 1
+        # Wait until we have nodes
+        whenUnsatisfiable: DoNotSchedule
+        labelSelector:
+          matchLabels:
+            # match on this helm chart install
+            app: fog-ingest
+            helm.sh/chart: {{ include "fogIngest.chart" . }}
+            {{- include "fogIngest.selectorLabels" . | nindent 6 }}
```

Comment:
Is this the correct indentation? Looks like it should be 12 or 14

Reply:
nope, good catch, I'm sure the build is just about to fail on that.